### PR TITLE
Fix sideNav for multiple transformers

### PR DESCRIPTION
**Pull request by jhonny218**
## Description
The window.sideNav is missing in the CMS build for the following node types:

- event_listing
- full_width_banner_alert
- health_care_local_facility
- health_care_region_detail_page
- leadership_listing
- person_profile
- press_releases_listing
- story_listing
- vamc_operating_status_and_alerts

The problem was missing data expected in the liquid templates when there was a revolving entity

## Testing done
Locally by comparing the HTML builds

## Screenshots
SideNav

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/stages/build/process-cms-exports/transformers/node-event_listing.js [hunk 1]
```diff
@@ -36,13 +36,17 @@ const transform = (entity, { ancestors }) => ({
   fieldDescription: getDrupalValue(entity.fieldDescription),
   fieldIntroText: getDrupalValue(entity.fieldIntroText),
   fieldMetaTitle: getDrupalValue(entity.fieldMetaTitle),
-  fieldOffice:
-    entity.fieldOffice[0] &&
-    !ancestors.find(r => r.entity.uuid === entity.fieldOffice[0].uuid)
-      ? {
-          entity: entity.fieldOffice[0],
+  /* eslint-disable no-nested-ternary */
```
Comment (Clark-W-Griswold) [inline src/site/stages/build/process-cms-exports/transformers/node-event_listing.js:39]: I wonder if there is a cleaner way to write this?

File: src/site/stages/build/process-cms-exports/transformers/node-health_care_local_facility.js [hunk 2]
```diff
@@ -57,11 +57,16 @@ const transform = (entity, { ancestors }) => ({
     entity.fieldOperatingStatusMoreInfo,
   ),
   fieldPhoneNumber: getDrupalValue(entity.fieldPhoneNumber),
-  fieldRegionPage:
-    entity.fieldRegionPage[0] &&
-    !ancestors.find(r => r.entity.uuid === entity.fieldRegionPage[0].uuid)
+  /* eslint-disable no-nested-ternary */
```
Comment (Clark-W-Griswold) [inline src/site/stages/build/process-cms-exports/transformers/node-health_care_local_facility.js:60]: Same as above.

**Review by Clark-W-Griswold — COMMENTED** [review]
What are our thoughts on the nested ternary and if we want to clean that up at all?

File: src/site/stages/build/process-cms-exports/transformers/node-event_listing.js (hunk 1 above)
Comment (jhonnyoddball) [reply src/site/stages/build/process-cms-exports/transformers/node-event_listing.js:39]: Any thoughts?

File: src/site/stages/build/process-cms-exports/transformers/node-health_care_local_facility.js (hunk 2 above)
Comment (jhonnyoddball) [reply src/site/stages/build/process-cms-exports/transformers/node-health_care_local_facility.js:60]: any suggestions here @U-DON ?

File: src/site/stages/build/process-cms-exports/transformers/node-health_care_local_facility.js [hunk 3]
```diff
@@ -57,11 +57,16 @@ const transform = (entity, { ancestors }) => ({
...
+  fieldRegionPage: entity.fieldRegionPage[0]
+    ? !ancestors.find(r => r.entity.uuid === entity.fieldRegionPage[0].uuid)
       ? { entity: entity.fieldRegionPage[0] }
-      : null,
+      : {
+          entity: {
+            title: getDrupalValue(entity.fieldRegionPage[0].title),
+          },
+        }
+    : null,
```
Comment (U-DON) [inline src/site/stages/build/process-cms-exports/transformers/node-health_care_local_facility.js:69]: Not a whole lot better, but something, I guess.

```suggestion
  fieldRegionPage: entity.fieldRegionPage[0]
    ? {
        entity: !ancestors.find(r => r.entity.uuid === entity.fieldRegionPage[0].uuid)
          ? entity.fieldRegionPage[0]
          : { title: getDrupalValue(entity.fieldRegionPage[0].title }
      }
    : null,
```

Comment (jhonnyoddball) [reply src/site/stages/build/process-cms-exports/transformers/node-health_care_local_facility.js:69]: @Clark-W-Griswold @U-DON Should I change them all this way?

Comment (U-DON) [reply src/site/stages/build/process-cms-exports/transformers/node-health_care_local_facility.js:69]: If you think it's cleaner, go for it.

**Review by U-DON — APPROVED** [review]

**Pull request merged**